fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -330,32 +322,32 @@ private RdfStream getResourceTriples(final int limit, final FedoraResource resou
             // Additional server-managed triples about this resource
             // Mementos already have the server managed properties in the PROPERTIES category
             // since mementos are immutable and these triples are no longer managed
-             if (ldpPreferences.prefersServerManaged() && !resource.isMemento()) {
-                streams.add(getTriples(resource, SERVER_MANAGED));
+            if (ldpPreferences.prefersServerManaged() && !resource.isMemento()) {
+                //streams.add(getTriples(resource, SERVER_MANAGED));
```

Comment:
I think include/omit choices were supposed to occur in a service...can't remember which service. So I agree this needs a `TODO` but I think it could be removed.

Reply:
will do.

Reply:
Done